.github/workflows/test.yml

```diff
@@ -26,9 +26,36 @@ jobs:
         script: |
             core.setFailed('Tagged commit does not match main')
 
-  test:
+  version-bump:
     runs-on: ubuntu-latest
     needs: [validate]
+    env:
+      VERSION: $GITHUB_REF_NAME
```

Comment:
I don't think this works as you expect. It's ok to use `${{ }}` syntax here as this string isn't interpreted second time later.

Reply:
So `${{ }}` syntax is dangerous in places where the resulting string going to be evaluated for the second time (e.g. shell scripts)

Reply:
Ahhh I see. Thank you for taking a look 🙏